intern/rest.py

```diff
@@ -0,0 +1,85 @@
+from rest_framework import filters
+from rest_framework import viewsets
+from rest_framework import status
+from rest_framework.permissions import DjangoModelPermissions
+from rest_framework.response import Response
+
+from intern.models import *
+from intern.serializers import InternRoleFullSerializer, InternSerializer, AccessLevelSerializer, InternGroupSerializer, \
+    RoleSerializer, AddInternRoleSerializer
+
+
+class InternViewSet(viewsets.ModelViewSet):
+    permission_classes = (DjangoModelPermissions,)
+    search_fields = ('user__username', 'user__realname')
+    filter_backends = (filters.SearchFilter, filters.DjangoFilterBackend)
+    filter_fields = ('user', 'user__username')
+
+    def get_queryset(self):
```

Comment:
Well. I think it just bloats the code, and there is no reason to have it when it is not needed.

Anyways, with Django 1.9 we either need `serializer_class` or set `ordering_fields` (see #41), so it would be nice to avoid having to change this when rebasing #41